### DISCUSSION TITLE
How to get the customization done in Adempiere Desktop in Adempiere Web Version

### DISCUSSION THREAD

**Discussion by amirsaifi** [adempiere/adempiere#4033] [Q&A] [answered]
Hi ADempiere
I have made some customization in ADempiere Payment window (in Interface and functionality both). How can I get the same in Web Version. Please guide.

**Comment by marcalwestf** [comment, marked as answer]
hi @amirsaifi in ADempiere, as of now, you have to implement UI for both ZK and Swing separately.
If you program it correctly, you'd only need to implement functionality once.

**Reply by amirsaifi** [reply]
Thnx Marcalwestf. Got it!.
